missed_events_replayer/missed_events_replayer/fetch_missing_matomo_requests.py

```diff
@@ -108,29 +86,79 @@ def run_query(start_timestamp, end_timestamp):
         | filter @logStream like /matomo-nginx/
         | filter status >= '500'
         | filter user_agent!='ELB-HealthChecker/2.0'
+        | filter user_agent!='Smokey Test'
         | filter path like /idsite=1/
         | filter path like /rec=1/""",
         limit=MAX_REQUESTS
     )
+    queryId = response['queryId']
+    status = 'Running'
+    seconds_slept = 0
+    while status != 'Complete':
+        time.sleep(1)
+        seconds_slept += 1
+        if seconds_slept % 30 == 0:
+            get_logger().debug(f'Still waiting for a request. Spent {seconds_slept} seconds waiting so far.')
+        response = client.get_query_results(queryId=queryId)
+        try:
+            status = response['status']
+        except KeyError:
+            get_logger.exception(f"'status' not found in response: {respons.keys()}")
```

Comment:
```suggestion
            get_logger.exception(f"'status' not found in response: {response.keys()}")
```
Tiny typo - might cause an issue trying to report what happened if you get a KeyError.

Reply:
Good spot